index.d/fabric8-services.yaml

```diff
@@ -0,0 +1,14 @@
+Projects:
```

Comment:
This entry has been removed. This introduces inconsistency with current state of index.